packages/ra-ui-materialui/src/input/TranslatableInputs.tsx

```diff
@@ -88,6 +98,7 @@ export const TranslatableInputs = (
                         groupKey={groupKey}
                         variant={variant}
                         margin={margin}
+                        {...rest}
```

Comment:
I find it weird that the TranslatableInputs extra props get pass down to such an "inner" component. In react-admin, we more commonly pass them to the root component. Instead of passing rest props, why don't you pass a `StackProps` prop?